src/signing.js

```diff
@@ -7,6 +7,8 @@
 
 var actualMethods = {};
 
+export var fastSigning;
```

Comment:
> export statments can be only occur at the top of the file.

That's not true.  AFAIK, _import_ statements can only occur at the top of the file.

> Setting fastSigning value depends on the flow of the program.

declared functions (as opposed to function expressions) can be called just fine during the initial evaluation of a script.  The definitions are hoisted above the execution of the script.  The following, for example, works:

```js

export const foo = load();

function load() {
    return "hello"
}

export const blah = 3;

console.log(foo);
console.log(blah);
```